pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -391,7 +392,7 @@ func newFrameworkWithQueueSortAndBind(r Registry, profile config.KubeSchedulerPr
 	if len(profile.Plugins.Bind.Enabled) == 0 {
 		profile.Plugins.Bind.Enabled = append(profile.Plugins.Bind.Enabled, config.Plugin{Name: bindPlugin})
 	}
-	return NewFramework(r, &profile, opts...)
+	return NewFramework(r, &profile, wait.NeverStop, opts...)
```

Comment:
Maybe pass a stopCh in `newFrameworkWithQueueSortAndBind` and defer to close the channel in the caller.